internal/pkg/template/templates/workloads/partials/cf/state-machine-definition.json.yml

```diff
@@ -13,7 +13,11 @@
       "Resource": "arn:${Partition}:states:::ecs:runTask.sync",
       "Parameters": {
         "LaunchType": "FARGATE",
+        {{- if eq .Platform.OS "WINDOWS_SERVER_2019_CORE" "WINDOWS_SERVER_2019_FULL"}}
```

Comment:
Do we need to do this for `fargate-taskdef-base-properties` as well?

Reply:
I wonder if we should move this logic into the `template` pkg so that we can write tests against it:
```yaml
"PlatformVersion": {{.Platform.Version}}
```

```go
package template

func (p RuntimePlatformOpts) Version() string {
    if strings.HasPrefix(p.OS, "WINDOWS_SERVER") { return "1.0.0" }
    return "LATEST"
}
```

Reply:
Quoting PH from the design doc, "Using LATEST might not be a good idea since if they update the latest platform version and there are unexpected incompatibilities not caught by our e2e tests and we did the release it could be a problem..."

Reply:
> Do we need to do this for `fargate-taskdef-base-properties` as well?

Yes. And it's also hard-coded here: https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/aws/ecs/ecs.go#L265. I'd had them separated because they seemed like separate logic streams, but I'll look at doing them now.